docs/design.md

```diff
@@ -67,8 +68,8 @@ NiceDay app. We will make use of Niceday trackers to gather data,
 and make use of the Niceday daily planner to plan activities.
 
 
-### Sensor data REST API
-A Sensor REST API that allows for the sensor data collection app to send data to PerfectFit system.
+### Sensor Data Handler
+A service that pulls data from the Sensor Data Database when needed.
```

Comment:
"pushes and pulls"? I guess all incoming data goes via this component which does validation, (additional) preprocessing, and sends to db. But the algo container could request data via this component too, right?